example/memory.py

```diff
@@ -4,6 +4,7 @@
 from functools import wraps
 from pathlib import PureWindowsPath
 
+import structlog
```

Comment:
winfspy doesn't use structlog, so better also not using (and stick to vanilla logging module) it in the example to avoid annoying import error

Reply:
Done!